spec/defines/watch_spec.rb

```diff
@@ -4,60 +4,42 @@
 describe 'consul_template::watch', :type => :define do
   context 'supported operating systems' do
     on_supported_os.each do |os, facts|
-
       context "on #{os}" do
         let(:facts) do
           facts.merge({
             :kernel => 'Linux',
-            :staging_http_get => 'curl'
+            :operatingsystem => os,
```

Comment:
Shouldn't `:kernel` and `:operatingsystem` already  be available from rspec-puppet-facts?